docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+   ```
+
+4. Install the `external-dns-management` component in your Namespace:
+
+   ```bash
+   helm install external-dns {PATH_TO_EXTERNAL_DNS_MANAGEMENT}/charts/external-dns-management --namespace={NAMESPACE_NAME} --set configuration.identifier=external-dns-identifier
+   ```
+
+5. Install the `cert-management` component in your Namespace:
+
+   ```bash
+   helm install cert-manager {PATH_TO_CERT_MANAGEMENT}/charts/cert-management --namespace={NAMESPACE_NAME} --set configuration.identifier=cert-manager-identifier
+   ```
+
+When you meet the prerequisites, go directly to step 2.
+
+## Steps
+
+Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
+
+1. Install Kyma 2.0 or higher.
+
+2. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+3. Create a Secret containing credentails for your DNS cloud service provider account. See the [official External DNS Management docuemntation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Once the YAML file with the required parameters is ready, create a Secret Custom Resource (CR). Run the following command:
+
+   ```bash
+   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+   ```
+
+4. Set up the `external-dns-management` component.
+
+- Create a DNSProvider CR. See the following example and modify values for the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters. As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     type: your-provider-dns-type # Edit this value
+     secretRef:
+       name: secret-name # Edit this value
+     domains:
+       include:
+         - mydomain.com # The main domain. Edit this value
+   EOF  
+   ```
+
+- Create a DNSEntry CR. See the following example and modify values for the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
```

Comment:
This annotation makes it to NOT work on GKE. The object is ignored. It's probably something to trigger the controller on `seed` cluster.

Reply:
As above.